internal/backends/postgresql/collection.go

```diff
@@ -73,14 +77,22 @@ func (c *collection) Query(ctx context.Context, params *backends.QueryParams) (*
 		}, nil
 	}
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/3414
 	q := fmt.Sprintf(
 		`SELECT %s FROM %s`,
```

Comment:
Let's extract that part into a common function too, because we will change it soon